lib/osf-components/addon/components/file-browser-item/component.ts

```diff
@@ -34,7 +36,9 @@ import moment from 'moment';
  * @class file-icon
  */
 @localClassNames('file-browser-item')
-export default class FileBrowserItem extends Component {
+export default class FileBrowserItem extends Component.extend({ styles }) {
```

Comment:
Do styles not work as a class property here?

Reply:
Yes, I put a note in the PR description

> Components that use `@localClassName` on properties need to have the style in the prototype (otherwise it breaks)